src/docs/truffle-dashboard/quickstart.md

```diff
@@ -123,15 +113,45 @@ module.exports = {
   networks: {
     // ... rest of network config
 
-    'truffle-dashboard': {
-      url: "http://localhost:24012/rpc"
-    }
+    "truffle-dashboard": {
+      url: "http://localhost:24012/rpc",
+    },
   },
 };
 ```
 
-From there, it can be used with any Hardhat task or tools like hardhat-deploy.
+### Connection
+
+Once you've added Truffle Dashboard as a network in your `hardhat.config.js`, it can be used with any Hardhat task or tools like hardhat-deploy.
 
 ```
 hardhat deploy --network truffle-dashboard
 ```
+
+## Decode requests
+
+Truffle Dashboard supports decoding requests for certain RPC calls, allowing you to view a more human-friendly
+representation of the messaging between your computer and the Ethereum network.
+
+<p class="alert alert-info">
+<i class="far fa-info-circle"></i> <strong>Information</strong>: The decode functionality is available
+in Truffle projects for the following methods: <code>eth_sendTransaction</code>, <code>personal_sign</code>, <code>eth_signTypedData_v3</code>, and
+<code>eth_signTypedData_v4</code>. We plan to support non-Truffle tooling such as Hardhat in the future.
+</p>
+
+Each time you run `truffle compile`, Truffle shares information about your contracts with Truffle Dashboard.
```

Comment:
There is currently a bug using `truffle compile` with Dashboard. Sometimes Truffle closes before the compilations can be saved. It sometimes works, however I think running `truffle migrate` in the Truffle development console while Dashboard is open is the way to guarantee they get saved.